### DISCUSSION TITLE
Discussion: Handle filtering by PROJECT PROPERTIES in the aggregated content web part

### DISCUSSION THREAD

**Discussion by tarjeieo** [Puzzlepart/prosjektportalen365#478] [Ideas]
**Is your feature request related to a problem? Please describe.**
The aggregated view web part (#394) is a great addition to PP365. Immediately the question came up from a customer: Can we in a way filter a view on project properties - not properties on the items themselves?

**Describe the solution you'd like**
Interested in how this can be achieved? Do we need two queries - one to filter on projects and one to filter on item of type XXX?

**Comment by olemp** [comment]
@tarjeieo We would need to separate queries I guess. Then based on the result from the projects query (with some filter on project properties) we need to filter what projects the items are retrieved from. I think it's doable, but it's a bit advanced.
